specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/SynapseLinkWorkspaces.json

```diff
@@ -0,0 +1,161 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-05-01-preview",
+    "title": "SqlManagementClient",
```

Comment:
API version for all resource types under a namespace need to be revised together due to some SDK and documentation generation dependencies

Reply:
@rkmanda , can you elaborate more on this. I didnt understand what needs to be done.

Reply:
@rkmanda , All required tests are passing now. Can you please clarify what ARM change is required here , you mentioned API version of resourcetypes ?

Reply:
roopesh was referring to the fact that all the various resources in Microsoft.SQL namespace need to be revised together , but that is fine I think your team is creating the dev-sql-Microsoft.Sql-2022-05-01-preview branch to enable this. You can ignore that comment